spec/queries/email_archive_query_spec.rb

```diff
@@ -48,7 +48,39 @@
       end
     end
 
-    context "when an email is not associated with content changes" do
+    context "when an email is associated with messages" do
+      let!(:subscriber) { create(:subscriber) }
+      let!(:email) { create(:archivable_email, subscriber_id: subscriber.id) }
+      let!(:subscription_contents) do
```

Comment:
Minor but I don't think the contents of this let really make sense due to both subscription contents being for the same subscriber associated with the same email which wouldn't happen in reality unless I'm misunderstanding something?

Reply:
I think the scenario this can occur in is a digest, where you have multiple messages associated with the same email and same subscriber.

Reply:
Oh okay, I always forget about the digest stream. 🤦‍♂